De-couple v8 and converged vr-tests and storybooks

ecraig12345 opened the pull request:
#### Pull request checklist

- [x] Include a change request file using `$ yarn change`

#### Description of changes

A few converged screener stories were still using things from v8, which is less than ideal for various reasons, and most of the usage was unnecessary. This PR de-couples the converged screener stories from all v8 libraries. Most of the changes involved switching from v8 styling helpers to just using the `style` prop.

One part of this change that's technically breaking but unlikely to cause problems is that I removed the `AvatarExamples` constant from `@fluentui/example-data` and put it inline with the Avatar screener stories, since that's the only place it's used. Usage of `@fluentui/example-data` should be basically limited to people who have used "export to codepen" on one of the examples from the v8 website (or copied it into a codesandbox), and since there are no converged Avatar examples on the website, that const should be unused.

I also de-coupled `@fluentui/storybook` (v8) from `@fluentui/react-storybook` (converged). This required duplicating the `withStrictMode` decorator, which isn't ideal, but also not terrible since it's extremely simple.

fabricteam commented:
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/131c59c96177944b46ac63905a592f22e20ec67c'>131c59c96177944b46ac63905a592f22e20ec67c</a></sub>
<div id="bundleSizeComment9423"/>

fabricteam commented:
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Avatar-mount_base.html">1040</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Avatar-mount.html">1059</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/BaseButton-mount_base.html">1097</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/BaseButton-mount.html">1092</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Breadcrumb-mount_base.html">3053</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Breadcrumb-mount.html">2951</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ButtonNext-mount_base.html">695</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ButtonNext-mount.html">694</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Checkbox-mount_base.html">1753</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Checkbox-mount.html">1701</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/CheckboxBase-mount_base.html">1496</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/CheckboxBase-mount.html">1491</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ChoiceGroup-mount_base.html">5251</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ChoiceGroup-mount.html">5161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ComboBox-mount_base.html">1166</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ComboBox-mount.html">1184</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/CommandBar-mount_base.html">11422</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/CommandBar-mount.html">11448</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ContextualMenu-mount_base.html">9288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ContextualMenu-mount.html">9443</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DefaultButton-mount_base.html">1286</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DefaultButton-mount.html">1367</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DetailsRow-mount_base.html">4141</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DetailsRow-mount.html">4137</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DetailsRowFast-mount_base.html">4083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DetailsRowFast-mount.html">4084</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DetailsRowNoStyles-mount_base.html">3978</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DetailsRowNoStyles-mount.html">3958</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Dialog-mount_base.html">2841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Dialog-mount.html">2873</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DocumentCardTitle-mount_base.html">294</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/DocumentCardTitle-mount.html">298</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Dropdown-mount_base.html">3557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Dropdown-mount.html">3529</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FluentProviderNext-mount_base.html">1977</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FluentProviderNext-mount.html">1956</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FluentProviderWithTheme-mount_base.html">248</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FluentProviderWithTheme-mount.html">311</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">211</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FluentProviderWithTheme-virtual-rerender.html">211</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">322</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">314</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FocusTrapZone-mount_base.html">2060</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FocusTrapZone-mount.html">2105</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FocusZone-mount_base.html">2139</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/FocusZone-mount.html">2081</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/IconButton-mount_base.html">1990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/IconButton-mount.html">2041</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Label-mount_base.html">520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Label-mount.html">478</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Layer-mount_base.html">3301</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Layer-mount.html">3319</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Link-mount_base.html">669</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Link-mount.html">632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/MakeStyles-mount_base.html">2083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/MakeStyles-mount.html">1991</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/MenuButton-mount_base.html">1683</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/MenuButton-mount.html">1691</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/MessageBar-mount_base.html">2268</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/MessageBar-mount.html">2247</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Nav-mount_base.html">3576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Nav-mount.html">3561</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/OverflowSet-mount_base.html">1309</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/OverflowSet-mount.html">1333</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Panel-mount_base.html">2794</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Panel-mount.html">2728</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Persona-mount_base.html">1025</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Persona-mount.html">1044</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Pivot-mount_base.html">1598</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Pivot-mount.html">1644</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/PrimaryButton-mount_base.html">1454</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/PrimaryButton-mount.html">1560</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Rating-mount_base.html">8120</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Rating-mount.html">8170</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/SearchBox-mount_base.html">1549</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/SearchBox-mount.html">1520</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Shimmer-mount_base.html">2754</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Shimmer-mount.html">2812</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Slider-mount_base.html">2196</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Slider-mount.html">2197</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/SpinButton-mount_base.html">5374</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/SpinButton-mount.html">5449</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Spinner-mount_base.html">583</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Spinner-mount.html">571</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/SplitButton-mount_base.html">3484</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/SplitButton-mount.html">3474</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Stack-mount_base.html">663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Stack-mount.html">685</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/StackWithIntrinsicChildren-mount_base.html">1879</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/StackWithIntrinsicChildren-mount.html">1847</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/StackWithTextChildren-mount_base.html">5031</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/StackWithTextChildren-mount.html">4958</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/SwatchColorPicker-mount_base.html">11199</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/SwatchColorPicker-mount.html">11144</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/TagPicker-mount_base.html">2947</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/TagPicker-mount.html">2964</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/TeachingBubble-mount_base.html">14160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/TeachingBubble-mount.html">14174</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Text-mount_base.html">581</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Text-mount.html">563</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/TextField-mount_base.html">1582</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/TextField-mount.html">1602</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ThemeProvider-mount_base.html">1388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ThemeProvider-mount.html">1375</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ThemeProvider-virtual-rerender_base.html">787</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ThemeProvider-virtual-rerender.html">746</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2055</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Toggle-mount_base.html">1016</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/Toggle-mount.html">988</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/buttonNative-mount_base.html">252</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test/buttonNative-mount.html">281</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
FlexMinimalPerf.default | [330](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/FlexMinimalPerf.default.html) | [308](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.07:1
RadioGroupMinimalPerf.default | [487](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [466](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.05:1
DatepickerMinimalPerf.default | [6490](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DatepickerMinimalPerf.default.html) | [6264](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1.04:1
HeaderMinimalPerf.default | [396](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/HeaderMinimalPerf.default.html) | [381](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.04:1
ListMinimalPerf.default | [567](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ListMinimalPerf.default.html) | [546](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.04:1
PortalMinimalPerf.default | [190](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/PortalMinimalPerf.default.html) | [182](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.04:1
SliderMinimalPerf.default | [1774](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/SliderMinimalPerf.default.html) | [1714](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1.04:1
TableMinimalPerf.default | [436](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TableMinimalPerf.default.html) | [421](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.04:1
FormMinimalPerf.default | [434](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/FormMinimalPerf.default.html) | [422](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.03:1
HeaderSlotsPerf.default | [808](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/HeaderSlotsPerf.default.html) | [786](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.03:1
ListCommonPerf.default | [689](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ListCommonPerf.default.html) | [669](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ListCommonPerf.default_base.html) | 1.03:1
PopupMinimalPerf.default | [652](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/PopupMinimalPerf.default.html) | [636](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.03:1
ReactionMinimalPerf.default | [413](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ReactionMinimalPerf.default.html) | [402](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.03:1
TooltipMinimalPerf.default | [1108](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TooltipMinimalPerf.default.html) | [1074](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.03:1
AnimationMinimalPerf.default | [443](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AnimationMinimalPerf.default.html) | [434](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.02:1
AttachmentMinimalPerf.default | [172](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AttachmentMinimalPerf.default.html) | [168](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.02:1
ButtonOverridesMissPerf.default | [1794](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1767](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 1.02:1
CarouselMinimalPerf.default | [511](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/CarouselMinimalPerf.default.html) | [500](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.02:1
ChatMinimalPerf.default | [699](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ChatMinimalPerf.default.html) | [686](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.02:1
CheckboxMinimalPerf.default | [2848](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2788](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.02:1
DialogMinimalPerf.default | [795](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DialogMinimalPerf.default.html) | [779](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.02:1
DropdownMinimalPerf.default | [3185](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DropdownMinimalPerf.default.html) | [3134](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.02:1
LayoutMinimalPerf.default | [394](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/LayoutMinimalPerf.default.html) | [385](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.02:1
LoaderMinimalPerf.default | [723](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/LoaderMinimalPerf.default.html) | [708](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.02:1
MenuButtonMinimalPerf.default | [1766](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1731](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.02:1
SegmentMinimalPerf.default | [376](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/SegmentMinimalPerf.default.html) | [368](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.02:1
SkeletonMinimalPerf.default | [379](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/SkeletonMinimalPerf.default.html) | [370](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.02:1
IconMinimalPerf.default | [665](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/IconMinimalPerf.default.html) | [649](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.02:1
TreeMinimalPerf.default | [862](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TreeMinimalPerf.default.html) | [846](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.02:1
TreeWith60ListItems.default | [192](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TreeWith60ListItems.default.html) | [189](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.02:1
BoxMinimalPerf.default | [368](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/BoxMinimalPerf.default.html) | [366](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.01:1
CardMinimalPerf.default | [601](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/CardMinimalPerf.default.html) | [595](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [726](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DropdownManyItemsPerf.default.html) | [718](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
MenuMinimalPerf.default | [910](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/MenuMinimalPerf.default.html) | [901](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.01:1
ProviderMinimalPerf.default | [1224](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ProviderMinimalPerf.default.html) | [1209](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.01:1
SplitButtonMinimalPerf.default | [4478](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [4428](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.01:1
StatusMinimalPerf.default | [737](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/StatusMinimalPerf.default.html) | [729](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.01:1
TableManyItemsPerf.default | [2005](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TableManyItemsPerf.default.html) | [1987](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1.01:1
AvatarMinimalPerf.default | [216](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AvatarMinimalPerf.default.html) | [216](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1:1
ButtonSlotsPerf.default | [576](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ButtonSlotsPerf.default.html) | [578](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1:1
EmbedMinimalPerf.default | [4249](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/EmbedMinimalPerf.default.html) | [4266](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1:1
GridMinimalPerf.default | [369](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/GridMinimalPerf.default.html) | [369](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/GridMinimalPerf.default_base.html) | 1:1
InputMinimalPerf.default | [1359](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/InputMinimalPerf.default.html) | [1364](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/InputMinimalPerf.default_base.html) | 1:1
ItemLayoutMinimalPerf.default | [1252](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [1250](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1:1
TextAreaMinimalPerf.default | [524](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TextAreaMinimalPerf.default.html) | [525](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1:1
CustomToolbarPrototype.default | [4465](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/CustomToolbarPrototype.default.html) | [4482](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1:1
RefMinimalPerf.default | [269](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/RefMinimalPerf.default.html) | [271](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/RefMinimalPerf.default_base.html) | 0.99:1
ToolbarMinimalPerf.default | [994](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ToolbarMinimalPerf.default.html) | [1007](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 0.99:1
AlertMinimalPerf.default | [288](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AlertMinimalPerf.default.html) | [294](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AlertMinimalPerf.default_base.html) | 0.98:1
ChatWithPopoverPerf.default | [390](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ChatWithPopoverPerf.default.html) | [400](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.98:1
LabelMinimalPerf.default | [402](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/LabelMinimalPerf.default.html) | [409](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/LabelMinimalPerf.default_base.html) | 0.98:1
ListNestedPerf.default | [592](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ListNestedPerf.default.html) | [605](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ListNestedPerf.default_base.html) | 0.98:1
DividerMinimalPerf.default | [371](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DividerMinimalPerf.default.html) | [381](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/DividerMinimalPerf.default_base.html) | 0.97:1
ImageMinimalPerf.default | [375](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ImageMinimalPerf.default.html) | [388](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ImageMinimalPerf.default_base.html) | 0.97:1
ProviderMergeThemesPerf.default | [1798](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1848](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 0.97:1
AttachmentSlotsPerf.default | [1111](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AttachmentSlotsPerf.default.html) | [1155](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 0.96:1
ListWith60ListItems.default | [665](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ListWith60ListItems.default.html) | [693](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ListWith60ListItems.default_base.html) | 0.96:1
AccordionMinimalPerf.default | [165](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AccordionMinimalPerf.default.html) | [173](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.95:1
ButtonMinimalPerf.default | [181](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ButtonMinimalPerf.default.html) | [191](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 0.95:1
ChatDuplicateMessagesPerf.default | [321](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [339](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 0.95:1
VideoMinimalPerf.default | [665](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/VideoMinimalPerf.default.html) | [700](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/VideoMinimalPerf.default_base.html) | 0.95:1
RosterPerf.default | [1182](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/RosterPerf.default.html) | [1277](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/RosterPerf.default_base.html) | 0.93:1
TextMinimalPerf.default | [349](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TextMinimalPerf.default.html) | [379](https://fluentuipr.z22.web.core.windows.net/pull/20907/perf-test-northstar/TextMinimalPerf.default_base.html) | 0.92:1

</details><div id="perfComment9423"/>

andrefcdias reviewed (commented):
LGTM but noticed Screener is failing. Is it that weird bug or consequence from the change?

ecraig12345 commented:
> LGTM but noticed Screener is failing. Is it that weird bug or consequence from the change?

It's consequences of the change, mostly correct.
- Accordion: changed to a different icon which renders slightly differently
- Avatar:
  - Most of these are expected because I added space between the avatars in some of the states. (I think there was always supposed to be space, but it wasn't working due to the FluentProvider being added by a decorator the immediate child of the element where the flex gap styles were applied.) 
  - The Avatar color changes are an unintended difference because I forgot to set a flex direction (I'll fix that)
- Divider: I accidentally changed the story name (that was a leftover from the original larger version of this change), which I'll fix

ecraig12345 commented:
Updated to fix the styling of the Avatar color stories, and switched the Divider stories back to their original name. I also made some intentional changes to the Divider stories to make the states easier to see.

ling1726 commented on apps/vr-tests/src/stories/AccordionConverged.stories.tsx:
```diff
@@ -7,7 +7,7 @@ import {
   AccordionPanel,
 } from '@fluentui/react-accordion';
-import { CircleRingIcon } from '@fluentui/react-icons-mdl2';
+import { Circle20Regular } from '@fluentui/react-icons';
```
might be better like this since, we'll have to move to unsize icons at some point.

```suggestion
import { Circle20Regular as CircleRingIcon } from '@fluentui/react-icons';
```

ling1726 commented on packages/storybook/src/decorators/withStrictMode.tsx:
```diff
@@ -0,0 +1,11 @@
+  return strictMode ? <React.StrictMode>{props.children}</React.StrictMode> : <>{props.children}</>;
+};
+
+export const withStrictMode = (storyFn: () => React.ReactNode) => {
```
I'm not entirely sure about duplicating this, since v9 vr tests still run on converged, so duplicating (and potentially diverging) SB infra could be a risk. I would prefer if we only duplicated once we break v9 vr-tests completely. I won't block your PR on this though

ecraig12345 replied:
The withStrictMode decorator may be present in vr-tests due to config inheritance, but it's never enabled. It wouldn't be useful in vr-tests anyway since it only logs console warnings.
